js/dashboard-demo/tts.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Created by mgl on 03.06.19.
+ */
+const textToSpeechLimit = 30;
+var socket;
+
+/**
+ *@function connect Connects the user html page to the server, connects the client to WebSocket, subscribes client to notifications
```

Comment:
Did you use the auto formatting or formatted manually?
- there is no space after * symbol
- the line is too long, it should be broken into multiple lines